README.md

```
@@ -1,25 +1,188 @@
+
+## Contributing
+
+Contributions welcome! Please read the existing code and documentation before submitting PRs.
+
+## License
+
+MIT License
+
+## Links
+
+- **Website:** https://bitcoinlink.app
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)